spec/models/user_spec.rb

```diff
@@ -367,4 +366,26 @@
       it { expect(subject.admin?).to be_truthy }
     end
   end
+
+  describe '#full_name' do
+    let(:user) { build(:user, first_name: 'first', last_name: 'last') }
+
+    context '@first_name=first @last_name=last' do
+      specify { expect(user.full_name).to eq('first last') }
+    end
+  end
+
+  describe 'Validations' do
```

Comment:
I'm not sure how closely this matches what you had in mind; let me know if I should update it.

Reply:
@patmbolger can certainly correct me, but based on his comment above, I think he just wants to be sure to test the 
```
validates_presence_of :first_name, :last_name
```

statement in the User model.  If you look at the tests for the other user models, you can see that this is what the `describe 'Validations' do...` blocks are used for.

Reply:
That is much simpler.